internal/core/cloud/aws_test.go

```diff
@@ -24,6 +25,7 @@ func TestAWSMetadataTestSuite(t *testing.T) {
 }
 
 func (suite *AWSMetadataTestSuite) TestNewAWSMetadata() {
+	ctx := context.TODO()
```

Comment:
Is this `TODO` the preferred practice for tests?

Reply:
Yep, it's an "empty context" made just to satisfy the interface

You could also use test context but it's not the preferred way, it's like mixing things